### PR TITLE
add Occt

### PR DESCRIPTION
**Pull request by looooo**
continued from here: https://github.com/conda-forge/staged-recipes/pull/1713
the build on windows will take about 55 min (64, and 32) but should work now. Feel free to interrupt if there are too many other builds waiting...

### CODE REVIEW

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -1,70 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (dopplershift) [inline recipes/example/meta.yaml]: Please don't remove the example recipe.

Comment (scopatz) [inline recipes/example/meta.yaml]: Yes, it is really necessary.  Removing the example breaks CI for all of staged recipes.

File: recipes/occt/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  version: 7.0.0
+
+source:
+  fn: occ.tar.gz
+  url: https://github.com/3drepo/occt/archive/d7355a4c5b7cac441e2c8821e369798266376a56.tar.gz
+  sha256: 1000486ada9dc48e766215c70e38982612780a9a7ccb113fd403aa207b8fabdc
+
+build:
+  number: 0
+  skip: true                              # [win and not py35]
```
Comment (scopatz) [inline recipes/occt/meta.yaml]: I think the style is to not align the selectors. Please fix this up in the feedstock

**Comment by looooo** [comment]
thanks!

**Pull request merged**